tests/disas.rs

```diff
@@ -459,14 +454,15 @@ where
             disassembly: disas,
         } in disas(bytes, sym.address())?.into_iter()
         {
-            if write_offsets || (prev_jump && !is_jump) {
-                write!(result, "{address:>4x}: ")?;
-            } else {
-                write!(result, "      ")?;
+            for (i, line) in disas.lines().enumerate() {
+                if i == 0 && (write_offsets || (prev_jump && !is_jump)) {
```

Comment:
It seems like we're trying to only have the targets of jumps be prefixed by address labels but then the test output ends up printing labels for everything after `0x33`?